packages/tools/devtools/devtools-core/src/data-visualization/VisualTree.ts

```diff
@@ -33,14 +33,14 @@ export enum VisualNodeKind {
 }
 
 /**
- * Type union representing TypeScript primitives.
+ * Type union representing TypeScript primitives supported in DDSes.
  *
  * @remarks Used for data / metadata in {@link VisualNodeBase}s.
  *
  * @internal
  */
 // eslint-disable-next-line @rushstack/no-new-null
-export type Primitive = bigint | number | boolean | null | string | symbol | undefined;
+export type Primitive = number | boolean | null | string | undefined;
```

Comment:
@Josmithr you're more familiar with this bit of devtools; any thoughts?

Reply:
Checked with @Josmithr while developing; those others should not be needed as they aren't (shouldn't be) used in DDSes as not serializable (at least not without special logic).
Please do review.